flow/scenarios/minicity.py

```diff
@@ -415,19 +415,16 @@ def specify_edges(self, net_params):
                  ]
 
         for edge in edges:
-            edge['numLanes'] = str(edge['numLanes'])
             if 'shape' in edge:
                 edge['length'] = sum(
                     [np.sqrt((edge['shape'][i][0] - edge['shape'][i+1][0])**2 +
                              (edge['shape'][i][1] - edge['shape'][i+1][1])**2)
                      * SCALING for i in range(len(edge['shape'])-1)])
-                edge['length'] = str(edge['length'])
-                edge['shape'] = ' '.join('%.2f,%.2f' % (blip*SCALING,
-                                                        blop*SCALING)
-                                         for blip, blop in edge['shape'])
+                edge['shape'] = [(blip*SCALING, blop*SCALING)
```

Comment:
uh, can we change this from blip and blop hahaha

Reply:
haha, fixed